airflow/providers/amazon/aws/transfers/mysql_to_s3.py

```diff
@@ -60,6 +60,10 @@ class MySQLToS3Operator(BaseOperator):
     :type index: str
     :param header: whether to include header or not into the S3 file
     :type header: bool
+    :param file_format: the destination file format, only csv and parquet are supported.
+    :type file_format: str
+    :param pd_parquet_kwargs: arguments to include in pd.to_parquet
+    :type pd_parquet_kwargs: dict
```

Comment:
We don't really need a dedicated kwargs per `file_format`.
We can deprecate `pd_csv_kwargs` have a generic `pandas_kwargs` similar to what we have in [hive](https://github.com/apache/airflow/blob/da99c3fa6c366d762bba9fbf3118cc3b3d55f6b4/airflow/providers/apache/hive/hooks/hive.py#L372)

Reply:
Andy guidance to keep backward compatibility? Like keeping pd_csv_kwargs but mark it deprecated?

Reply:
Updated